flake.nix

```diff
@@ -1,7 +1,8 @@
 {
   description = "The purely functional package manager";
 
-  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05-small";
+  #inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05-small";
+  inputs.nixpkgs.url = "github:edolstra/nixpkgs/fix-aws-sdk-cpp";
```

Comment:
https://nixpk.gs/pr-tracker.html?pr=256102 once this hits small, we can go back.